webpack-demo/src/index.js

```diff
@@ -1,61 +1,41 @@
 import './style.css';
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] Kindly make sure to refrain from using eslint disable as it defies the purpose of using linters in the first place.